### DISCUSSION TITLE
Change app url for Default Password reset email template

### DISCUSSION THREAD

**Discussion by qxygene1** [pocketbase/pocketbase#6107] [Q&A]
How to change {APP_URL} from js api?

```
        await pb
          .collection('users')
          .requestPasswordReset("test@email.com", { app_url: ? });
```

**Comment by ganigeorgiev** [comment]
Changing it from the client-side is insecure and doesn't really make sense.

The default email template can be changed only though the specific collection settings options from the Dashboard or via the hooks.

**Comment by qxygene1** [comment]
This could be a useful feature for managing multiple websites from a single panel.

**Reply by ganigeorgiev** [reply]
As mentioned in our communication earlier on the support mail, please try to elaborate more on your use case and why what you are asking is needed in the first place so that other users can eventually assist you.

For me personally I don't understand your use case and it seems insecure to allow redirecting users with the password reset token to random/client-provided urls. If you really want to do it you can implement it via the [`onRecordRequestPasswordResetRequest`](https://pocketbase.io/docs/js-event-hooks/#onrecordrequestpasswordresetrequest) and/or [`onMailerBeforeRecordResetPasswordSend`](https://pocketbase.io/docs/js-event-hooks/#onmailerrecordpasswordresetsend) hooks, but again I don't recommend it.

**Comment by qxygene1** [comment]
https://github.com/pocketbase/js-sdk

// Sends a record password reset email.
🔓 pb.collection(collectionIdOrName).requestPasswordReset(email, options = {});

Can we pass variables? 

At Default Password reset email template it says Available placeholder parameters:  {APP_NAME} {APP_URL} {RECORD:*} {TOKEN}

Can you explain  {RECORD:*}  how to use this?  {RECORD:*variable} ?

**Reply by chnfyi** [reply]
You can't pass variables with `pb.collection(collectionIdOrName).requestPasswordReset(email, options = {})`. Options argument can be used to provide custom headers or custom fetch options.

In your password reset email template you can only use fields you have configured for the collection: `{RECORD:id}`.

**Reply by qxygene1** [reply]
Thank you, can you provide an example of using RECORD:id please.

**Reply by chnfyi** [reply]
Of course. Assuming that you configured a `name` field for your auth collection, you can try:

```html
<p>Hello, {RECORD:name}</p>
<p>Click on the button below to reset your password.</p>
<p>
  <a class="btn" href="{APP_URL}/_/#/auth/confirm-password-reset/{TOKEN}" target="_blank" rel="noopener">Reset password</a>
</p>
<p><i>If you didn't ask to reset your password, you can ignore this email.</i></p>
<p>
  Thanks,<br/>
  {APP_NAME} team
</p>
```
